src/components/graph/Graph.jsx

```diff
@@ -101,7 +101,7 @@ export default class Graph extends React.Component {
 
         const forceLink = d3ForceLink(this.state.d3Links)
             .id(l => l.id)
-            .distance(D3_CONST.LINK_IDEAL_DISTANCE)
+            .distance(this.state.config.link.size || D3_CONST.LINK_IDEAL_DISTANCE)
             .strength(D3_CONST.FORCE_LINK_STRENGTH);
```

Comment:
What do you think of also having this constants configurable:

```javascript
// Some d3 constant values
const D3_CONST = {
    FORCE_LINK_STRENGTH: 1,
    LINK_IDEAL_DISTANCE: 100,
    SIMULATION_ALPHA_TARGET: 0.05
};
```

They are on the top of this file